docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -60,8 +60,10 @@ cloud providers is difficult.
 {% capture prerequisites %}
 
 1. One or more machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+
-1. 1GB or more of RAM per machine (any less will leave little room for your
+1. 2GB or more of RAM per machine (any less will leave little room for your
    apps)
+1. 2CPU or more on the master
+1. Approximately 10GB free disk space on the master
```

Comment:
Where does this come from?

Reply:
This could even be low. The base files themselves seem to take up 5GB. In a single-host install, I experienced reproducible failures with a 10GB drive. Troubleshooting with Casey, we determined it was a lack of free hard disk space. After bumping it up to 20GB, it was smooth sailing for me. We recommend 15GB in the Calico docs, but I put it at 10GB here because I figured it might be controversial. I would actually recommend someone start with 15GB minimum if they want to complete this install.